src/helpers/turbo.ts

```diff
@@ -0,0 +1,92 @@
+import { capture } from '@snapshot-labs/snapshot-sentry';
+import snapshot from '@snapshot-labs/snapshot.js';
+import db from './mysql';
+import { fetchWithKeepAlive } from './utils';
+
+type Space = {
+  id: string;
+  turbo_expiration: number;
+};
+
+const SCHNAPS_API_URL = process.env.SCHNAPS_API_URL;
+const NETWORK_PREFIX = process.env.NETWORK === 'mainnet' ? 's:' : 's-tn:';
+const RUN_INTERVAL = 10 * 1e3; // 10 seconds
+
+// Periodically sync the turbo status of spaces with the schnaps-api
+export async function trackTurboStatuses() {
+  if (!SCHNAPS_API_URL) return;
+
+  while (true) {
+    // Step 1: Query all the spaces from the schnaps-api
+    let spaces = await getSpacesExpirationDates();
+
+    spaces = spaces
+      .filter(space => space.id.startsWith(NETWORK_PREFIX))
+      .map(space => {
+        return { ...space, id: space.id.replace(NETWORK_PREFIX, '') };
+      });
+
+    // Step 2: Update the turbo status of the spaces in the database
+    updateTurboStatuses(spaces);
+
+    // Sleep for a while before running the loop again
+    await snapshot.utils.sleep(RUN_INTERVAL);
+  }
+}
+
+async function updateTurboStatuses(spaces: { id: string; turbo_expiration: number }[]) {
+  if (spaces.length === 0) return;
+
+  // Sync `turbo_expiration` for each space
+  const query = `
+    UPDATE spaces
+    SET
+      turbo_expiration = CASE id
+        ${spaces.map(() => `WHEN ? THEN ?`).join(' ')}
+      END
+    WHERE id IN (${spaces.map(() => '?').join(', ')});
```

Comment:
Can't we just use this?
```suggestion
    WHERE id IN (?);
```